src/scale.ts

```diff
@@ -144,5 +144,5 @@ export interface Scale {
   /**
    * Uses the source data range as scale domain instead of aggregated data for aggregate axis. This option does not work with sum or count aggregate as they might have a substantially larger scale range.
```

Comment:
Update to this option only works with ... <list of function or pointer to the const storing all valid aggregation function for use raw domain> ...